### ISSUE TITLE
Linux package

### ISSUE THREAD

**Issue by trymeouteh** [tecnovert/basicswap#19]
Please release this app in one of the following linux package types...

AppImage
Flatpak

**Comment by peepo5** [comment]
I agree. The barrier to entry is currently too high. Having verified builds would be great, as those not tech-savvy would likely struggle installing.

**Comment by detherminal** [comment]
I agree too. If we want more people to use DEX'es, we should make it much easier to jump in.

**Comment by seasharpfoss** [comment]
Hi Everyone,

 I'd like to help out by investigating how to build something that requires less craftiness to get going.

My understanding of flatpak and appimage is pretty basic. It sounds like we're going to need:

- A way to run basicswap-prepare and basicswap-run from the same appimage/flatpak
- A simple setup process that presumes friendly defaults and thereby automates the crafty steps

Let me sort out what's possible as far as these two needs go, and please share your thoughts in the meantime.

**Comment by seasharpfoss** [comment]
I've started hacking on the electron build. I've updated the [basicswap-installation-GUI issue](https://github.com/gerlofvanek/basicswap-installation-GUI/issues/24) with [the errors I'm grappling with](https://github.com/gerlofvanek/basicswap-installation-GUI/issues/24#issuecomment-2099550940)

What might help is if someone can test the electron app that I'm currently able to build, which will make sure the tools I'll eventually use for the flatpak are working correctly. Can anyone help with that perhaps?

I'll update the thread once I have more information.

**Comment by mexwei** [comment]
Yes please make appimage to make easy for users.

**Comment by nahuhh** [comment]
https://github.com/nahuhh/basicswap-bash

you can get up and running using this

**Comment by mexwei** [comment]
> https://github.com/nahuhh/basicswap-bash
> 
> you can get up and running using this

You not understand, it not same. It not an appimage or flatpak. They very easy for user. They will bring more users
